### ISSUE TITLE
Gem installation can prompt for stdin causing an infinite loop

### ISSUE THREAD

**Issue by rtyler** [jruby-gradle/jruby-gradle-plugin#65]
In my particular case having two gems which install the same binstub:

```
Overwrite the executable? [yN]  protobuffy's executable "protoc-gen-ruby" conflicts with protobuf
Overwrite the executable? [yN]  protobuffy's executable "protoc-gen-ruby" conflicts with protobuf
Overwrite the executable? [yN]  protobuffy's executable "protoc-gen-ruby" conflicts with protobuf
Overwrite the executable? [yN]  protobuffy's executable "protoc-gen-ruby" conflicts with protobuf
Overwrite the executable?
> Building 0% > :jrubyPrepareGems
```

**Comment by rtyler** [comment]
This may be solved by #159

**Comment by rtyler** [comment]
I am going to close this, I don't have a reproduction case anymore. :man_shrugging:
